_contentTemplates/common/get-started.md

```diff
@@ -1,85 +1,44 @@
-#blazor-tutorial-intro
-## Step 1 - Set Up a Blazor Project
+#prerequisites-download
+>tip This step-by-step tutorial starts with the basics. If you are already familiar with the Telerik NuGet Feed and Blazor in general, you may prefer the [Telerik UI for Blazor workflow]({%slug getting-started/what-you-need%}) article.
 
+## Prerequisites
 
->Make sure that you have installed the following:
-* .NET - [.NET Core 3.1.x](https://dotnet.microsoft.com/download/dotnet-core/3.1), [.NET 5](https://dotnet.microsoft.com/download/dotnet/5.0) or [.NET 6](https://dotnet.microsoft.com/download/dotnet/6.0).
-* Visual Studio - [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) (for .NET 3.x and .NET 5) or [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) (for .NET 6).
->
->The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}}`, and it supports `{{site.supportedFrameworkVersion}}`.
-
-#end
+To successfully complete the steps in this tutorial, make sure that you have installed the following:
 
+* <a href="https://dotnet.microsoft.com/en-us/download/dotnet" target="_blank">.NET Core 3.1.x</a> or later
 
+* <a href="https://visualstudio.microsoft.com/vs/" target="_blank">Visual Studio 2019</a> or later
```

Comment:
Better refer to the System Requirements page (`[...]({%slug system-requirements%})`) to avoid hard-coding versions here. We don't list Visual Studio versions there, but...
* VS versions are coupled with .NET versions anyway.
* VS is not required to use our components. If a .NET developer doesn't have VS, they will be aware how to perform the described steps in another way, so no need to over-explain here.